plugins/header_rewrite/value.h

```diff
@@ -59,6 +60,11 @@ class Value : Statement
     } else {
       s += _value;
     }
+    if (s.find("$") != std::string::npos && _regex_cond != nullptr) {
```

Comment:
Wouldn't it be better to check if _regex_cond is available first? Before searching the string?